### ISSUE TITLE
Problems with 1.0

### ISSUE THREAD

**Issue by Aaron1178** [SRombauts/UE4GitPlugin#25]
Hello, my team and I tried to use the Git plugin with no prevail. Is the required process to have github for windows open at the same time Unreal Engine 4 is open, and inside unreal engine 4, when a file has been added, then go to the git for windows UI and sync it to the master branch? or should you not have to even touch git for windows? 

Your advice would be greatly appreciated.

Thanks,

**Comment by SRombauts** [comment]
Yes, currently the Git plugin is only for local work, commit and diff
viewing, but does not know about the remote server.

Le mar. 3 mars 2015 11:28, Aaron Thompson <notifications@github.com> a
écrit :

**Comment by Aaron1178** [comment]
Hello, ok, so our team are thinking of volunteering myself to contribute to this git plugin. I will fork the repo and scan through everything to catch up if that's ok with you? What are the next steps in order to get the plugin working for multi developer support?

**Comment by SRombauts** [comment]
That would be nice.
You should in fact fork the whole Unreal Engine on GitHub, as my Git plugin
had now been integrated directly into the Engine!

There you could see how to use the 'sync' operation to fetch or pull from
the default git remote.

Cheers!

Le mer. 4 mars 2015 00:16, Aaron Thompson <notifications@github.com> a
écrit :

**Comment by Aaron1178** [comment]
I already have the unreal engine source ;)

**Comment by SRombauts** [comment]
What I mean is that any improvement should go to the official Epic
UnrealEngine repository through GitHub pull request.

Le mer. 4 mars 2015 07:36, Aaron Thompson <notifications@github.com> a
écrit :

**Comment by SRombauts** [comment]
Push/Pull is now partly supported for Git LFS 2 Lock workflow
